From 0430e51224ee618c733e4657ec94a2093793f59b Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Wed, 30 Jun 2010 20:18:08 -0400
Subject: [PATCH 2/3] [terminal] Add OPOST to tty attributes

ONLCR apparently requires OPOST to work.

This should prevent weird "stair stepping"
effects when non-plymouth things try to
use the terminal.
---
 src/libply-splash-core/ply-terminal.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/libply-splash-core/ply-terminal.c b/src/libply-splash-core/ply-terminal.c
index 028551d..fb6a29d 100644
--- a/src/libply-splash-core/ply-terminal.c
+++ b/src/libply-splash-core/ply-terminal.c
@@ -186,7 +186,7 @@ ply_terminal_set_unbuffered_input (ply_terminal_t *terminal)
   term_attributes.c_iflag |= ICRNL;
 
   /* Make \n return go to the beginning of the next line */
-  term_attributes.c_oflag |= ONLCR;
+  term_attributes.c_oflag |= ONLCR | OPOST;
 
   if (tcsetattr (terminal->fd, TCSANOW, &term_attributes) != 0)
     return false;
-- 
1.6.5.2

